lib/logstash/outputs/redis.rb

```diff
@@ -92,6 +92,9 @@ class LogStash::Outputs::Redis < LogStash::Outputs::Base
   # Only supported for `list` Redis `data_type`.
   config :congestion_threshold, :validate => :number, :default => 0
 
+	# Auto-Queue switching when full, instead of waiting.
```

Comment:
This doc is a bit confusing to me, here's an alternative:

"If multiple redis servers are configured and congestion is detected, switch to a different redis server."